src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -64,6 +66,7 @@ public Measure createVersion(String id, String versionType, String username, Str
     measure.getMeasureMetaData().setDraft(false);
     measure.setLastModifiedAt(Instant.now());
     measure.setLastModifiedBy(username);
+    measure = setMeasureReviewMetaData(measure);
```

Comment:
Reassignment can be skipped as Java will mutate the passed in object directly.

Reply:
@jkotanchik-SB measure here is the object from db, the ReviewMetaData could be null, so I have a special method setMeasureReviewMetaData() to set the ReviewMetaData.